tasks/patternpack.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (grunt) {
     build: "./html",
     src: "./src",
     assets: "./src/assets",
-    theme: npmPath + "patternpack-example-theme",
+    theme: "patternpack-example-theme",
```

Comment:
The enhanced theme configuration will now accept a package name.  This is a simplified configuration that takes advantage of the new capabilities